tests/glsl-intrinsic/atomic/atomicCounterTestMultiple.slang

```diff
@@ -17,15 +17,15 @@ void computeMain()
 {
 
     outputBuffer.data[0] = true
-// CHECK_GLSL-DAG: one_0._data_0[3]
+// CHECK_GLSL: one_0._data_0[3]
```

Comment:
I see this is where "_data" takes an effect.
But is it compatible to GLSL syntax?
I mean when we compile this test with a following command, would the output shader cause any syntax error with glslc?
```
slangc.exe -allow-glsl -stage compute -entry computeMain -target glsl -DTARGET_GLSL atomicCounterTestMultiple.slang -o test.glsl
glslc.exe test.glsl
```

Reply:
Can you confirm if it works in a way I am worried about?

Reply:
the way we force the use of atomic_uint does work; the compute test shows this

the multiCounter test does not run a compute test since same bindings are not managed by the current testing infrastructure and did not change it since this is the only time it would be useful